db/openshift.deploy.yml

```diff
@@ -326,3 +329,95 @@ objects:
               dnsPolicy: "ClusterFirst"
               serviceAccountName: "${JOB_SERVICE_ACCOUNT}"
               serviceAccount: "${JOB_SERVICE_ACCOUNT}"
+  - kind: CronJob
+    apiVersion: "batch/v1"
+    metadata:
+      name: ${NAME}-${ZONE}-${COMPONENT}-restore
+      labels:
+        app: ${NAME}-${ZONE}
+        cronjob: ${NAME}-${ZONE}
+    spec:
+      schedule: "0 0 31 2 *" # At 00:00 on day-of-month 31 in February, so it won't run, we'll trigger it manually when need
+      concurrencyPolicy: "Replace"
+      successfulJobsHistoryLimit: ${{SUCCESS_JOBS_HISTORY_LIMIT}}
+      failedJobsHistoryLimit: ${{FAILED_JOBS_HISTORY_LIMIT}}
+      jobTemplate:
+        metadata:
+          labels:
+            app: ${NAME}-${ZONE}
+            cronjob: ${NAME}-${ZONE}
+        spec:
+          backoffLimit: ${{JOB_BACKOFF_LIMIT}}
```

Comment:
Where are these limits set?

Reply:
these are set for the db backup cronjob, I thought maybe I can use them here as well

Reply:
Hmm, not sure if that makes sense, since we have retry = never.

Reply:
um... this I'm not so sure... just follow the one for db backup

Reply:
I changed to 0. The restart policy sets to never will ensure when the pod fails, it won't be restarted. But we still need to set backofflimit to make sure the job will not be retired.